### ISSUE TITLE
Unnecessary table announcement is announced by screen reader when navigated to the ‘Format’ control present in the dialog

### ISSUE THREAD

**Issue by HValluri27** [microsoft/fluentui#27300]
**Environment**:

OS: Windows 11 22H2 (OS Version 22621.819) 
Browsers: Edge Chromium (107.0.1418.52) (Official build) (64-bit)
Screen Readers: NVDA (2022.3.2)
 
**Accessibility Issue :**
 We don't have any table configured in our custom code, on selection/clicking of filter option we are getting table 0 rows 0 columns by screen reader, we don't have control on this to modify as it is default behavior rendering from Office -Ui-Fabric.
The filter panel makes use of microsoft's OOB UI component ( office-ui-fabric-react )  where a GroupedList from Fluent UI is used to make out the filter panel along with its children.

**Actual Result:**
Screen Reader is announcing unnecessary table information for the Format control,
Screen reader current announcement is observed as ‘table with 0 rows and 0 columns Format button collapsed’

**Expected Result:**
Screen Reader should not announce unnecessary table information for the Format control,
Screen reader current announcement is expected to be as ‘Format button collapsed’

**Comment by behowell** [comment]
Hi @HValluri27, in order to help us investigate this, please provide a codepen that reproduces the issue. You can start by clicking "Export to Codepen" on one of the examples in the documentation, if that helps: https://developer.microsoft.com/en-us/fluentui#/controls/web/groupedlist

**Comment by HValluri27** [comment]
Hi,
Apologies for delay reply.
Please find the below screenshot regarding the table announcement in https://developer.microsoft.com/en-us/fluentui#/controls/web/groupedlist,when screen reader announcement captured while navigating the filter section.
![image](https://user-images.githubusercontent.com/128687916/233269627-86a0d6ba-6d54-4fa2-89a7-fa9b071e4a72.png)
